packages/core/src/components/table/table-body-row/table-body-row.tsx

```diff
@@ -71,8 +71,8 @@ export class TdsTableBodyRow {
     cancelable: false,
     bubbles: true,
   })
+  // return clickevent
   tdsClick: EventEmitter<{
-    // return clickevent
     event: MouseEvent;
```

Comment:
Just an FYI, this could also be a KeyboardEvent. Since it can be "pressed" by space or enter while tabbing.